src/KeysManager.php

```diff
@@ -0,0 +1,164 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\pubkey_encrypt\KeysManager.
+ */
+
+namespace Drupal\pubkey_encrypt;
+
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\user\PrivateTempStoreFactory;
+use Drupal\user\UserInterface;
+
+/**
+ * Handles users' Public/Private key pairs.
+ */
+class KeysManager {
+  protected $entityTypeManager;
+  protected $tempStore;
+
+  /*
+   * Constructor with dependencies injected to it.
+   */
+  public function __construct(EntityTypeManagerInterface $entityTypeManager, PrivateTempStoreFactory $tempStoreFactory) {
+    $this->entityTypeManager = $entityTypeManager;
+    $this->tempStore = $tempStoreFactory->get('pubkey_encrypt');
+  }
+
+  /*
+   * Initialize all users' keys.
+   */
+  public function initializeAllUserKeys() {
+    $users = $this->entityTypeManager->getStorage('user')->loadMultiple();
+
+    foreach ($users as $user) {
+      $this->initializeUserKeys($user);
+    }
+  }
+
+  /*
+   * Initialize a specific user's keys.
+   */
+  public function initializeUserKeys(UserInterface $user) {
+    // Generate a Public/Private key pair.
+    $config = array(
+      "config" => "C:/xampp/apache/bin/openssl.cnf",
```

Comment:
This is a windows and environment-specific path. This won't work on other machines.